cpp/harvest/.gitignore

```diff
@@ -2,3 +2,4 @@ core
 marc_issn_lookup
 nacjd
 remid_augment
+core_issn_lookup
```

Comment:
Please don't forget moving the cc file and the .gitignore entry from `cpp/harvest/core_issn_lookup.cc` to `cpp/issn_lookup.cc`